README.md

```diff
@@ -102,23 +102,30 @@ SC-55mk2-v1.01/waverom2.bin      4d91cdeaed048d653dbf846a221003c3a3f08279
 
 The main build method is via CMake and vcpkg. This is what the CI workflow uses.
 
-Alternatively, you can use [Meson](#meson-alternative-build-method), but that build method is less frequently tested.
+On Linux you can also build without vcpkg, using system libraries.
 
 ### Prerequisites
 
 #### All platforms
 
 - CMake (3.27.0 or later)
-- vcpkg (latest)
 
-#### Windows
+#### Windows (using vcpkg)
 
+- vcpkg (latest)
 - Visual Studio 2022 (17.11.0 or later)
 
-#### macOS or Linux
+#### macOS or Linux (using vcpkg)
+
+- vcpkg (latest)
+- Clang (16.0.0 or later)
+- Ninja (1.12.0 or later)
+
+#### Linux (using system libraries, without vcpkg)
 
 - Clang (16.0.0 or later)
 - Ninja (1.12.0 or later)
+- SpeexDSP (1.2.1 or later)
```

Comment:
Okay, so all this confuses the README a bit the way you've done it, and static building is still the standard method on all supported platforms for the plugin.

Can you leave the top part completely alone, just like it was, with only this single sentence added at the end of the `macOS or Linux` section?

```
See the [Using system libs on Linux (alternative build method)](link) section if you want dynamic linking.
```

Then add all the relevant info about using the `syslibs` presets in a new **Using system libs on Linux (alternative build method)** section at the end to compartmentalise it better.

Reply:
Done, but with one modification - I have put the information about alternative build method at the same place where it was. The `Using system libs on Linux (alternative build method)` is present just after the `Linux` section, so if the reader is at the end of `Linux` section, it is already well visible. Let me know what you think.

BTW, I did not touch the `Cleaning the build directory` - the diff detection got confused.